### DISCUSSION TITLE
2.Instanz URL /1/ fehlt

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2559] [General]
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/a3232f15-024c-4b11-91e5-df0ead56ef36)

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/a8936c98-3f7f-4260-89bd-ddfec695cbf6)

Und trotzdem wird die richtige config genutzt

**Comment by Zefau** [comment]
~~Die Instanz wird beim initialen Aufruf im Browser-Cache gespeichert und dann aus der URL entfernt.~~ Wurde geändert

**Comment by mcuiobroker** [comment]
Beim ersten Aufruf scheint es noch Probleme zu geben.
Wenn man den SKIP auslöst und dann mit Next weiter geht -> weiße Page.

-> man darf die erste Begrüßung nicht SKIPpen können?

Bricht man die automatische Generierung ab, erhält man das Menü links erst mit einem angelegten Gerät.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/04b4859d-027d-4e8e-934c-3b7186b7424b)

- Wenn man die Instanz löscht bekommt man keinen Hinweis auf dem Frontend. Nur bei F12 sieht man die Fehlermeldungen

- Es müsste wenigstens die Liste der  Tastenkombinationen und ein Hinweis auf die Doku angezeigt werden, falls man geskipped hat oder nichts in der Geräteliste steht
